tests/excerptexport/views/test_export_list.py

```diff
@@ -41,21 +42,20 @@ def setUp(self):
             orderer=User.objects.create_user('another_user', 'someone_else@example.com', 'top secret')
         )
 
-    def test_permission_denied_if_lacking_email(self, *args):
+    def test_redirect_to_login_if_not_logged_in(self, *args):
         response = self.client.get(
             reverse(
                 'excerptexport:export_list'
             )
         )
-        # redirect to 'Access Denied' page
-        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(response, '/login/?next=/exports/', fetch_redirect_response=False)
```

Comment:
By default, `assertRedirects` also tests whether fetching the target page gives a `200` status. Our tests settings are currently not configured so that our social login pages will work, thus I've skipped that step with `fetch_redirect_response=False`.